packages/lodestar/src/chain/bls/interface.ts

```diff
@@ -9,6 +9,12 @@ export type VerifySignatureOpts = {
    * Only non-time critical objects should be marked as batchable, since the pool may hold them for 100ms.
    */
   batchable?: boolean;
+
+  /**
+   * Use main thread to verify signatures, use this with care.
+   * Ignore the batchable option if this is true.
+   */
+  useMainThread?: boolean;
```

Comment:
Should we name `verifyOnMainThread` for consistency with `blsVerifyOnMainThread`?